### ISSUE TITLE
Reduce duplicated code using framework/library

### ISSUE THREAD

**Issue by ExpressHermes** [pulkithanda/Portfolio-hf#21]
A lot of the code is duplicated between the HTML files. It would be a good idea to use a framework/library to reduce this. We can use react/nodejs or anything else.

**Comment by pulkithanda** [comment]
@ExpressHermes Good idea but if you use backend, many people don't know how to run flask/node apps like in my voyager repository that becomes a problem. If you have a solution to this do tell me

**Comment by ExpressHermes** [comment]
@pulkithanda React.js or any other frontend library can be used if you using a backend framework is an issue. 

PS: I might not be able to work on this exclusively hence unassigning myself :)
